analyzers/tests/SonarAnalyzer.Test/Helpers/MemberDescriptorTest.cs

```diff
@@ -411,13 +410,13 @@ private static InvocationContext CreateContextForMethod(string typeAndMethodName
             Assert.Fail($"Test setup error: could not find method call in test code snippet: {typeAndMethodName}");
             return null;
 
-            IEnumerable<(SyntaxNode node, string name)> GetCSharpNodes() =>
-                snippet.GetNodes<CSharpSyntax.InvocationExpressionSyntax>()
-                    .Select(n => ((SyntaxNode)n, CSharpSyntaxNodeExtensions.GetIdentifier(n.Expression)?.ValueText));
+            IEnumerable<(SyntaxNode Node, string Name)> GetCSharpNodes() =>
```

Comment:
This is not raising T0003 (ValueTuple)?

Reply:
No, because it's a test code. See

https://github.com/SonarSource/sonar-dotnet/blob/c755a17547427db2d15e93c13a4e69ef04651134/analyzers/src/SonarAnalyzer.CSharp.Styling/Rules/AvoidValueTuple.cs#L26